### PR TITLE
Add a TS fixture and change tests to run it

### PR DESCRIPTION
**Pull request by Gregoor**
Heya,

so I only dropped a first TS fixture in and modified the test to run it. Hence this is a super tiny PR, wanted to do things more incremental this time so that I don't get lost in too grand things like before (I was reminded of the joke: How do you eat an elephant? One bite at a time).

Anyway, for my future help to be more useful then this, should I maybe work on `convert.rs` testing the whole suite of fixtures?

### CODE REVIEW

**Comment by Gregoor** [comment]
@dwoznicki, is this too minimal to be useful? I just wanted to get an early feel this time. I could work on a new PR which changes the test to run alle the fixtures?

**Comment by dwoznicki** [comment]
@Gregoor Haha, no this looks perfect for the current level. I just missed it (no email notification from GitHub). Shall I merge this or do you want to keep working from this PR?

Also, I'm going to be changing the structure of babel-compat a bit to improve compile time soon.

**Comment by Gregoor** [comment]
Okay so I did a potential bad thing or two. The fixtures function didn't appear to be running for me so I thought I could just hack away at it, inlining things, running rustfmt, renaming them with the only real change being that I put some branching in for detecting whether it's a TS fixture. Then I learned the reason it was not running originally was because IntelliJ can't handle that cool test-runner stuff yet and runs the fixture in a way that filters those tests out.

So TLDR: Here's a PR which only adds this small typescript-fixture inclusion thing but also bring with it all these other things. Let me know if that's not cool, because that's not definitely not how I'd send a work PR and I'll redo it with only the TS change.

On another note, I've got some more time tomorrow if there's any other way in which I can help with the babel conversion, let me know!

**Comment by dwoznicki** [comment]
Yeah, that's totally fine. The babel conversion is still a mess, so let's just try and get it working for now and worry about commit/PR organization later. If the babel-visit crate is working (fingers crossed), we should be able to write AST normalizers and begin to really test in earnest. I'm about to test, and I'll comment on https://github.com/swc-project/swc/pull/1465 if all goes well.
